src/inverted-index.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+class InvertedIndex {
+
+  /**
+   * class constructor
+   * @constructor
+   */
+  constructor() {
+    this.indexedFiles = this.indexedFiles || [];
```

Comment:
Since it's in the constructor, there is no need for `||` `this.indexedFiles` will always be undefined so you can just initialize it to `[]`